src/Compilers/CSharp/Test/IOperation/IOperation/IOperationTests_IVariableDeclaration.cs

```diff
@@ -4082,25 +4082,22 @@ void M()
     Statements (0)
     Next (Regular) Block[B1]
         Entering: {R1}
-
 .locals {R1}
```

Comment:
It doesn't look like there are meaningful changes in this file. Consider reverting